lang/po/cataclysm-dda.pot

```diff
@@ -94,7 +94,7 @@ msgstr ""
 msgid "plant fiber"
 msgstr ""
 
-#. ~ Description for plant fiber
+#. ~ Description for plant fibe
```

Comment:
Why are these comments missing the last character? (Could probably be related to my recent changes to make it output unix line endings on Windows, but I'm not sure how.)

Reply:
Probably yes. I removed the following code from `lang/update_pot.sh` and the output `cataclysm-dda.pot` is then correct.

```bash
# convert line endings to unix
if [[ $(uname -s) =~ ^(CYGWIN|MINGW)* ]]
then
    echo "> Converting line endings to Unix"
    if ! sed -i -e 's/\r$//' lang/po/cataclysm-dda.pot
    then
        echo "Line ending conversion failed. Aborting."
        exit 1
    fi
fi
```

Reply:
That's weird, it was working properly on my Windows 10. Also the regexp is supposed to remove '\r' at line end only... What platform were you running the script on?

Reply:
I'm running on macOS.